packages/core/src/actions/getPriceFromPriceReporter.ts

```diff
@@ -20,7 +20,7 @@ export async function getPriceFromPriceReporter(
   const {
     exchange = 'binance',
     base,
-    quote = DEFAULT_QUOTE[exchange].address,
+    quote = getDefaultQuotes()[exchange].address,
```

Comment:
nit: at this point probably could just pass exchange as the input like `getDefaultQuoteToken(exchange).address`